packages/cf-component-modal/example/basic/component.js

```diff
@@ -55,11 +55,26 @@ class Component extends React.Component {
   }
 }
 
+class Application extends React.Component {
+  render() {
+    return (
+      <GatewayProvider>
+        <div>
+          {this.props.children}
+          <GatewayDest name="modal"/>
+        </div>
+      </GatewayProvider>
+    );
+  }
+}
```

Comment:
Upgrading to the latest version of react-gateway is a breaking change for cf-component-modal